SaveImage.cpp

```diff
@@ -181,8 +181,6 @@ int __stdcall ObjectMemory::SaveImageFile(const char* szFileName, bool bBackup,
 			stream.setbuf(buf, sizeof(buf));
 
 			bSaved = SaveImage(stream, &header, nRet);
-			stream.close();
```

Comment:
stream will be closed by the destructor. Once a stdio stream has been _fdopen'd onto a low-level I/O descriptor, the descriptor must not be closed with _close, rather the stream should be closed with fclose (which will in turn _close fd)